### DISCUSSION TITLE
Got an old Flash Builder project, working on air 14 SDK. Where do I find it?

### DISCUSSION THREAD

**Discussion by danielkotzer** [airsdk/Adobe-Runtime-Support#3295] [General]
Hi,
I got an old FB project of an app, and was asked to make it work. Some features that were available on Air SDK 14, are not supported anymore and I'd like to make this app work, at least at the beginning, and then replace the unsupported component later on.
Is there some way I can get this version of Air SDK? Maybe Harman has it? how do I contact?
Thanks

**Comment by ajwfrost** [comment]
We can't provide AIR SDK 14 I'm afraid.. maybe best to start with something like 33.1.1.743 so that you keep the WebKit / HTMLLoader feature as it had been earlier? Otherwise the only feature I'm aware of to change is the H.264/AAC multimedia support..

If you don't have Flash Builder now, I think VS Code + the AS3/MXML extension does a pretty good job of importing the earlier project files into the asconfig.json format..
